ladybug/comfort/standard/comfortmodel.py

```diff
@@ -0,0 +1,24 @@
+# coding=utf-8
+"""Comfort standard base objects."""
+
+
+class ComfortModel(object):
+    """Thermal comfort Model base class."""
+
+    def __init__(self):
+        self._calc_length = 0
+        self._is_computed = True
+
+    @property
+    def isComfortModel(self):
```

Comment:
This is a bad practice that I started to address the type issue in Dynamo. I think we should remove this from all the code base at some point.

Reply:
Ah, but I have written checks that look for this attribute in order to identify the type of input.  What is the correct way to check the type of object, then?

Reply:
We discussed that the isinstance() check is the right way to do this and we will plan to implement this in a later PR.